### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,21 +20,14 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
...
- - `mir.las` [linear algebra subroutines](https://github.com/DlangScience/mir/issues/48)
- - `mir.stat` [statistical functions](https://github.com/DlangScience/mir/issues/49)
-  - `mir.stat.probcounting` hyperloglog algorithm implementation
+### In progress
 
+ - `mir.sparse` Sparse Tensors (see sparse branch and `v0.15.1-beta2`+)
+  -  `mir.sparse.blas` Spars BLAS
 
-## Notes
+## Documnetation
```
Comment (wilzbach) [inline README.md:28]: Typo

File: README.md [hunk 2]
```diff
@@ -1,18 +1,13 @@
 [![codecov.io](https://codecov.io/github/DlangScience/mir/coverage.svg?branch=master)](https://codecov.io/github/DlangScience/mir?branch=master)
-[![Build Status](https://travis-ci.org/DlangScience/mir.svg?branch=master)](https://travis-ci.org/DlangScience/mir)
-[![Build status](https://ci.appveyor.com/api/projects/status/ir2k3o3j0isqp7pw/branch/master?svg=true)](https://ci.appveyor.com/project/9il/mir/branch/master)
```
Comment (wilzbach) [inline README.md:3]: Why did you remove appveyor?

File: README.md [hunk 3]
```diff
@@ -58,6 +46,7 @@ Numeric library and mirror for upcoming numeric packages for the Dlang standard
 ### On any platform with dub
 
 [![Dub version](https://img.shields.io/dub/v/mir.svg)](http://code.dlang.org/packages/mir)
+[![Dub downloads](https://img.shields.io/dub/dt/mir.svg)](http://code.dlang.org/packages/mir)
```
Comment (wilzbach) [inline README.md:49]: Once the count gets higher, we probably want to put it back to the top?

**Comment by 9il** [comment]
Sorry that I have not waiting you! Needed to update readme before the answer http://forum.dlang.org/post/bkdjoplpibvzccbawkcw@forum.dlang.org

File: README.md (hunk 3 above)
Comment (9il) [inline README.md:49]: This is only about dub and I move it to dub. A lot of shields in one place look weird

File: README.md [hunk 4]
```diff
@@ -1,18 +1,13 @@
 [![codecov.io](https://codecov.io/github/DlangScience/mir/coverage.svg?branch=master)](https://codecov.io/github/DlangScience/mir?branch=master)
-[![Build Status](https://travis-ci.org/DlangScience/mir.svg?branch=master)](https://travis-ci.org/DlangScience/mir)
-[![Build status](https://ci.appveyor.com/api/projects/status/ir2k3o3j0isqp7pw/branch/master?svg=true)](https://ci.appveyor.com/project/9il/mir/branch/master)
-[![Circle CI](https://circleci.com/gh/DlangScience/mir.svg?style=svg)](https://circleci.com/gh/DlangScience/mir)
-
-[![Dub version](https://img.shields.io/dub/v/mir.svg)](http://code.dlang.org/packages/mir)
```
Comment (wilzbach) [inline README.md:6]: Hmm I think it's worth the duplication, current version should be visible to the user.

File: README.md (hunk 2 above)
Comment (9il) [inline README.md:3]: We already have it in the table

File: README.md (hunk 1 above)
Comment (9il) [inline README.md:28]: thx)

File: README.md (hunk 3 above)
Comment (wilzbach) [inline README.md:49]: Yep I understood :)

File: README.md (hunk 2 above)
Comment (wilzbach) [inline README.md:3]: Ok - fair enough :)

File: README.md (hunk 4 above)
Comment (9il) [inline README.md:6]: But this should be the last stable version. Not my sparse branch =/

**Comment by wilzbach** [comment]
> Sorry that I have not waiting you! Needed to update readme before the answer

No worries - you couldn't know that I am still awake and it's just a trivial change ;-)

File: README.md (hunk 4 above)
Comment (wilzbach) [inline README.md:6]: Agreed - is there no github tag shield?
